ignite/metrics/gan/fid.py

```diff
@@ -0,0 +1,218 @@
+import warnings
+from distutils.version import LooseVersion
+from typing import Callable, Optional, Sequence, Union
+
+import numpy as np
+import torch
+
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = [
+    "FID",
+]
+
+
+def fid_score(
+    mu1: torch.Tensor, mu2: torch.Tensor, sigma1: torch.Tensor, sigma2: torch.Tensor, eps: float = 1e-6
+) -> Union[torch.Tensor, float]:
```

Comment:
@gucifer finally, I think it would be better to fix the output type to float and not between torch tensor or float. Let's explicitly call `.item()` and convert it to float.